examples/scrap_clipboard.py

```diff
@@ -57,7 +57,7 @@ def usage():
                 elif r is None:
                     print(f"Type {t} : None")
                 else:
-                    print("Type {} : '{}'".format(t, r.decode("ascii", "ignore")))
+                    print(f"Type {t} : '{r.decode("ascii", "ignore")}'")
```

Comment:
this is a syntax error... as I pointed in an above review. Droid suggested a workaround that works too (replacing the nested `"` with `'`)

Reply:
```suggestion
                    print(f"Type {t} : '{r.decode('ascii', 'ignore')}'")
```